Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,16 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const durationInHours = tasks.map(minute => minute.duration / 60);
+  const moreThan2Hours = durationInHours.filter(newArray => newArray >= 2);
+  const wageForTotalHours = moreThan2Hours.map(wage => wage * hourlyRate);
```

Comment:
We not mapping wages here, we are mapping durations. So the parameter name `wage` is incorrect.